File: scala-play-server/app/model/ResponseGeocodingProperties.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ResponseGeocodingProperties.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2020-07-14T11:29:17.777Z[Etc/UTC]")
case class ResponseGeocodingProperties(
  name: String,
  label: String,
  score: Option[Double],
  houseNumber: Option[String],
  street: Option[String],
  region: Option[String],
  regionCode: Option[String],
  neighbourhood: Option[String],
  county: Option[String],
  macroregion: Option[String],
  city: Option[String],
  country: Option[String],
  countryCode: Option[String],
  continent: Option[String],
  postcode: Option[String],
  features: Option[ResponseMapInfoFeatures]
)

object ResponseGeocodingProperties {
  implicit lazy val responseGeocodingPropertiesJsonFormat: Format[ResponseGeocodingProperties] = Json.format[ResponseGeocodingProperties]
}

File: fsharp-functions/OpenAPI/src/model/ResponseGeocodingProperties.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open OpenAPI.Model.ResponseMapInfoFeatures

module ResponseGeocodingProperties = 

  //#region ResponseGeocodingProperties

  [<CLIMutable>]
  type ResponseGeocodingProperties = {
    [<JsonProperty(PropertyName = "name")>]
    Name : string;
    [<JsonProperty(PropertyName = "label")>]
    Label : string;
    [<JsonProperty(PropertyName = "score")>]
    Score : double;
    [<JsonProperty(PropertyName = "house_number")>]
    HouseNumber : string;
    [<JsonProperty(PropertyName = "street")>]
    Street : string;
    [<JsonProperty(PropertyName = "region")>]
    Region : string;
    [<JsonProperty(PropertyName = "region_code")>]
    RegionCode : string;
    [<JsonProperty(PropertyName = "neighbourhood")>]
    Neighbourhood : string;
    [<JsonProperty(PropertyName = "county")>]
    County : string;
    [<JsonProperty(PropertyName = "macroregion")>]
    Macroregion : string;
    [<JsonProperty(PropertyName = "city")>]
    City : string;
    [<JsonProperty(PropertyName = "country")>]
    Country : string;
    [<JsonProperty(PropertyName = "country_code")>]
    CountryCode : string;
    [<JsonProperty(PropertyName = "continent")>]
    Continent : string;
    [<JsonProperty(PropertyName = "postcode")>]
    Postcode : string;
    [<JsonProperty(PropertyName = "features")>]
    Features : ResponseMapInfoFeatures;
  }
  
  //#endregion
  